### ISSUE TITLE
Excel Slicer not accessible

### ISSUE THREAD

**Issue by Ronan555** [nvaccess/nvda#6605]
Hi,

Excel 2013  Slicers seem inaccessible to NVDA. Can this be investigated?

Many thanks,
Ronan

**Comment by jcsteh** [comment]
Can you please provide a spreadsheet containing an example of a slider
which is not accessible? Also, please provide the exact steps you took,
what you expected and what happened instead. Thanks.

**Comment by Ronan555** [comment]
[Sample Excel report with slicers.xlsx](https://github.com/nvaccess/nvda/files/627751/Sample.Excel.report.with.slicers.xlsx)
Hi,

Slicers are a feature of Excel that are used with Pivot Tables.
In Microsoft Excel 2010 and later , you have the option to use slicers to filter the data in a pivot table rather than traditional filters. Slicers provide buttons that you can click to filter PivotTable data. In addition to quick filtering, slicers also indicate the current filtering state, which is meant  to make it easy to understand what exactly is shown in a filtered PivotTable report.

I have attached a spreadsheet that contains slicers and a pivot table.

This is like a report but NVDA doesn't seem to recognise any content here.

I haven't created this myself. This is a sample of a type of report that I need to read.

Is there any way to access this with NVDA?
Thanks,
Ronan

**Comment by derekriemer** [comment]
Hi: This issue tracker is public. Is this document confidential? Just checking.

**Comment by Ronan555** [comment]
No, spreadsheet is not confidential. Thanks for reminding us of this important point.

**Comment by Adriani90** [comment]
Ok, tested with NVDA 2019.1 RC1 and Excel 2010 and Excel 2016 / 365.

STR:
1. Open the excel sheet provided by the initial author
2. Got to "salaries"
3. Press alt to open the ribbon
4. Press right arrow to go to the options of the pivot table
5. Press tab until you land on slizer (mihgt have a different name in Excel 2016)
6. Press enter on "insert a new slize"
or
7. Press enter on slizer manager

Excel 2010:
The dialogs are not accessible. The options in the list where you can choose a slizer is not being read.

Excel 2016 / 365:
When entering the dialog, NVDA begins to read all the items in the list of slizers without pressing anything. The items are accessible in the new slize dialog, but not in the slize manager.
In Office 2016 / 365, when pressing f6 to access the slizers, NVDA does not report anything.

cc: @lukaszgo1, @Qchristensen, @novalis7747.

**Comment by Qchristensen** [comment]
This is all still the same in Office 365 16.0.11601.20174 and NVDA 2019.1.1.

(Also, just to clarify, since I did the same thing for ages - Office 365 is not the same as Office 2016 or Office 2019.
